corehq/apps/userreports/pillow_utils.py

```diff
@@ -18,7 +32,15 @@ def rebuild_sql_tables(adapters):
         else:
             all_adapters.append(adapter)
     for adapter in all_adapters:
-        tables_by_engine[adapter.engine_id][adapter.get_table().name] = adapter
+        if not _is_datasource_active(adapter):
```

Comment:
Seems like this should be negated as well? The `else` branch says it encountered a "deactivated data source".
```suggestion
        if _is_datasource_active(adapter):
```

Reply:
I have been playing around with what the fn name if it should be `is_active` or `is_deactivated` and at the end refactored it in wrong way 😅